### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Luckyaremu**
#Javascript testing practice

-All errors fixed
-All five test completed
-All five test passed
-Readme well descriptive to run the test.

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @Luckyaremu and @kelibst 
You have done really well :clap: However, you need to fix a few issues before the project can be approved.
- [ ] Add linters and configure it, use GitHub actions [refer](https://github.com/microverseinc/linters-config/tree/master/javascript)
- [ ] The .eslintrc.json and .stylelintrc.json code is not the one used by microverse, refer from link above
### Make the changes and submit for another review. Good luck.
- Happy coding!

By Raymond
Slack @iamraymond

**Review by wathigo — APPROVED** [review]
### Approved :heavy_check_mark: 
Great work implementing the suggested changes.

**You may move on.**

**Good luck :four_leaf_clover: 
Simon Wathigo.**

**Comment by Luckyaremu** [comment]
#Testing practice.
-The pull request has been approved but on my dashboard and that of my partner, it is still showing changes required, Kindly help us do the needful. Warm regards
Untitled

File: functions/capitalize/capitalize.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const capitalize = require('./capitalize');
+
```
Comment (shloch) [inline functions/capitalize/capitalize.test.js:2]: Change this to `import` like you did in the other test files

File: jest.config.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+
+  clearMocks: true,
+
+
+};
```
Comment (shloch) [inline jest.config.js:9]: Change this to `export` or `export default`  etc. like you did in the other test files

File: functions/capitalize/capitalize.test.js (hunk 1 above)
Comment (shloch) [reply functions/capitalize/capitalize.test.js:2]: `require` isn't pure ES6, it's ndoejs

File: functions/analyze/analyze.test.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import analyze from './analyze';
+
+test('Gets average', () => {
+  expect(analyze([1, 8, 3, 4, 2, 6]).average).toEqual(4);
+});
+
+test('Gets minimum value', () => {
+  expect(analyze([1, 8, 3, 4, 2, 6]).min).toEqual(1);
+});
+
```
Comment (shloch) [inline functions/analyze/analyze.test.js:10]: ### [optional changes]
Try to add some tests that test negative scenario
exampe: analyze([1, 8, 3, 4, 2, 6]) to not equal 16

File: README.md [hunk 4]
```diff
@@ -1,2 +1,52 @@
...
+
+The main idea is simply to start working on your code by writing automated tests before writing the code that is being tested.
+
+## Setup
+
+- Clone the repository on your local machine
+- cd into the folder, find the index.html file and open it
+- npm install
+- npm test
+
```
Comment (shloch) [inline README.md:17]: - Write the Actual commands to clone <including the clone URL>, as well as the one to `cd` into directory
- Also put the commands to appear as commands by putting them into backtits characters (so that they can appear like this --> `code`)

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @Luckyaremu 

Great job...You tested your modules correctly, and all I get is green lights as shown
Screen Shot 2020-08-24 at 10 15 36 PM

I want you to change few import/exports as well as enrich your README a little as mentioned below (BEFORE MERGING)
The rest is really good

I will validate it..... but don't forget to work on what I just wrote

Happy coding
